assets/src/edit-story/components/panels/design/textStyle/textStyle.js

```diff
@@ -18,30 +18,45 @@
  * External dependencies
  */
 import { __ } from '@web-stories-wp/i18n';
-
+import { useRef } from 'react';
 /**
  * Internal dependencies
  */
 import getUpdatedSizeAndPosition from '../../../../utils/getUpdatedSizeAndPosition';
+import { styles, useFocusHighlight, states } from '../../../../app/highlights';
 import { SimplePanel } from '../../panel';
 import { usePresubmitHandler } from '../../../form';
 import StyleControls from './style';
 import ColorControls from './color';
 import FontControls from './font';
 
 function StylePanel(props) {
+  const fontDropdownRef = useRef(null);
+  const textColorRef = useRef(null);
+  // use highlights to update panel styles
+  // but don't dynamically adjust the `isPersistable` prop on `SimplePanel`
+  // the textStyle panel automatically opens already whenever a text element is selected
+  // if we update this to only be when there's a highlight the functionality that is expected
+  // will be wrong.
+  const dropdownHighlight = useFocusHighlight(states.FONT, fontDropdownRef);
+  const colorHighlight = useFocusHighlight(states.TEXT_COLOR, textColorRef);
+
   // Update size and position if relevant values have changed.
   usePresubmitHandler(getUpdatedSizeAndPosition, []);
 
   return (
     <SimplePanel
       name="textStyle"
       title={__('Text', 'web-stories')}
+      css={
+        (dropdownHighlight?.showEffect || colorHighlight?.showEffect) &&
+        styles.FLASH
+      }
       isPersistable={false}
     >
-      <FontControls {...props} />
+      <FontControls {...props} fontDropdownRef={fontDropdownRef} />
       <StyleControls {...props} />
-      <ColorControls {...props} />
+      <ColorControls {...props} textColorRef={textColorRef} />
```

Comment:
So I thought about it, and landed on this because it doesn't really matter here since it's getting passed from getting set up in `textStyle` passed through `ColorControls` to `Color` (which is shared and  currently has the forward ref). The ref isn't getting used at all in `ColorControls` so it feels unnecessary to set up forwardRef on the whole string. Same goes for the fontDropdownRef - that's going from `textStyle` passed through `FontControls` to `FontPicker` to the shared `AdvancedDropDown` where the ref is actually in play. 

I was trying to keep as many components that create the textStyle panel out of the forwarded refs as possible just to keep things as simple as possible.